### ISSUE TITLE
Make linked test cases point to the Internet Archive

### ISSUE THREAD

**Issue by timvandermeij** [mozilla/pdf.js#6854]
We have quite a number of linked test cases in the https://github.com/mozilla/pdf.js/tree/master/test/pdfs folder. Some of them point to sources other than the Internet Archive or Bugzilla and might become unavailable, which is to be avoided.

It is better to fetch the files from the Internet Archive to be sure that they remain available. This should always be possible if the current source exists because once you input the link on the Internet Archive website it will archive the file automatically.

Below is a list of linked test cases that point to external sources and should be updated (obtained with `grep -L "bugzilla.mozilla.org\|archive.org" *.pdf.link`). Make sure that the hash of the Internet Archive file is the same as the hash in the test manifest.

- [x] 20130226130259.pdf.link
- [x] JST2007-5.pdf.link
- [x] P020121130574743273239.pdf.link
- [x] SFAA_Japanese.pdf.link
- [x] TaroUTR50SortedList112.pdf.link
- [x] aboutstacks.pdf.link
- [x] artofwar.pdf.link
- [x] bpl13210.pdf.link
- [x] bug766138.pdf.link
- [x] bug808084.pdf.link
- [x] bug887152.pdf.link
- [x] cable.pdf.link
- [x] ecma262.pdf.link
- [x] fips197.pdf.link
- [x] fit11-talk.pdf.link
- [x] geothermal.pdf.link
- [x] hmm.pdf.link
- [x] html5checker.pdf.link
- [x] hudsonsurvey.pdf.link
- [x] ichiji.pdf.link
- [x] issue1010.pdf.link
- [x] issue1015.pdf.link
- [x] issue1049.pdf.link
- [x] issue1096.pdf.link
- [x] issue1127.pdf.link
- [x] issue1133.pdf.link
- [x] issue1155.pdf.link
- [x] issue1169.pdf.link
- [x] issue1233.pdf.link
- [x] issue1257.pdf.link
- [x] issue1309.pdf.link
- [x] issue1317.pdf.link
- [x] issue1419.pdf.link
- [x] issue1466.pdf.link
- [x] issue1597.pdf.link
- [x] issue1629.pdf.link
- [x] issue1658.pdf.link
- [x] issue1685.pdf.link
- [x] issue1687.pdf.link
- [x] issue1709.pdf.link
- [x] issue1721.pdf.link
- [x] issue1729.pdf.link
- [x] issue1796.pdf.link
- [x] issue1810.pdf.link
- [x] issue1878.pdf.link
- [x] issue1912.pdf.link
- [x] issue1936.pdf.link
- [x] issue1998.pdf.link
- [x] issue2006.pdf.link
- [x] issue2129.pdf.link
- [x] issue2139.pdf.link
- [x] issue2386.pdf.link
- [x] issue2442.pdf.link
- [x] issue2531.pdf.link
- [x] issue2537.pdf.link
- [x] issue2627.pdf.link
- [ ] issue2799.pdf.link
- [x] issue2829.pdf.link
- [x] issue2853.pdf.link
- [x] issue2881.pdf.link
- [x] issue3062.pdf.link
- [x] issue3384.pdf.link
- [x] issue3666.pdf.link
- [x] issue3848.pdf.link
- [x] issue3903.pdf.link
- [x] issue3925.pdf.link
- [x] issue3999.pdf.link
- [x] issue4926.pdf.link
- [x] issue5592.pdf.link
- [ ] issue5726.pdf.link
- [x] issue818.pdf.link
- [x] issue919.pdf.link
- [x] jai.pdf.link
- [x] kdchart.pdf.link
- [x] liveprogramming.pdf.link
- [x] mao.pdf.link
- [x] ocs.pdf.link
- [x] ohkubo-SS04.pdf.link
- [x] pal-o47.pdf.link
- [x] pdf.pdf.link
- [x] piperine.pdf.link
- [x] preistabelle.pdf.link
- [x] protectip.pdf.link
- [x] tcpdf_033.pdf.link
- [x] tutorial.pdf.link
- [x] txt2pdf.pdf.link
- [x] unix01.pdf.link
- [x] usmanm-bad.pdf.link
- [x] vesta.pdf.link
- [ ] yo01.pdf.link

Additionally:
- [x] Investigate `hmm.pdf`. The MD5 hash in the manifest matches the MD5 hash of the file from the link, but the bots complain about a mismatch.

**Comment by Snuffleupagus** [comment]
Regarding `issue1155`: I'm not having any luck in obtaining a working link from the Internet Archive. However, looking at the issue/PR (and quickly browsing through the structure of the PDF file), it seems that it shouldn't be too hard to hand-craft a reduced test-case, so I'll look into that later.

**Comment by Snuffleupagus** [comment]
It appears that we've now reached the point where every file, perhaps with one or two exceptions, that are possible to obtain through the Internet Archive have been fixed.
Below is a summary of the ones that are left to fix, so we probably need to start looking into the possibility of replacing them with reduced test-cases, if at all possible.
A number of these files are font related, which in *theory* should make them reasonable easy (albeit time-consuming) to create/verify reduced test-cases. (It might be simplesteasier to see if Brendan manages to create the tool https://github.com/mozilla/pdf.js/pull/6650#issuecomment-159081259  
```
test/pdfs/issue1597.pdf.link
test/pdfs/issue2537.pdf.link
test/pdfs/issue2627.pdf.link
test/pdfs/issue2799.pdf.link
test/pdfs/issue5726.pdf.link
test/pdfs/kdchart.pdf.link
test/pdfs/yo01.pdf.link
```
